herbert_core/applications/multifit/@mfclass/private/mask_data_for_fit.m

```diff
@@ -16,9 +16,9 @@
 %   wmask       Cell array (row) of masked datasets.
 %               If there was an error, wmask is an empty cell array size (1,0)
 %
-%   msk_out     Cell array (row) of mask arrays, one per data set.
+%   keep_out    Cell array (row) of mask arrays, one per data set.
 %               Same size as input data.
-%               If there was an error, msk_out is an empty cell array size (1,0)
+%               If there was an error, keep_out is an empty cell array size (1,0)
```

Comment:
It is not at all obvious why keep_out is a better name than mask out. How does keep/msk_out differ from msk_in? Does size here refer to numel(w) only or to sizes of pixel arrays?